src/cmd/build.rs

```diff
@@ -294,11 +297,14 @@ fn optimize_wasm(crate_metadata: &CrateMetadata) -> Result<OptimizationResult> {
 /// If successful, the optimized Wasm is returned as a `Vec<u8>`.
```

Comment:
This comment should be updated to say that this now writes the file to `dest_wasm`